src/backend/mailbox_manager/tests/api/mailboxes/test_api_mailboxes_create.py

```diff
@@ -305,6 +305,37 @@ def test_api_mailboxes__create_pending_mailboxes(domain_status):
     assert mailbox.status == "pending"
 
 
+def test_api_mailboxes__should_not_create_duplicate_email():
+    """
+    Should not create duplicate email when email already exists.
+    """
+    access = factories.MailDomainAccessFactory(
+        role=enums.MailDomainRoleChoices.ADMIN,
+        domain=factories.MailDomainEnabledFactory(),
+    )
+    existing_mailbox = factories.MailboxFactory(domain=access.domain)
+
+    client = APIClient()
+    client.force_login(access.user)
+    mailbox_values = dict(serializers.MailboxSerializer(existing_mailbox).data)
+    # import pdb; pdb.set_trace()
```

Comment:
Part of me is curious to know the story behind this line (and the one like it below). And part of me is afraid to hear it 😜

Reply:
I just wanted to make sure all info were the same but only the `local_part` field should matter. I'll simplify it right away